api/WEBGL_draw_buffers.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/WEBGL_draw_buffers",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "36"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": false
```

Comment:
The previous data was true. Are you sure it should be false, or is it not supported on some hardware?

Reply:
Reading https://developer.mozilla.org/en-US/docs/Web/API/WEBGL_draw_buffers reveals that this extension is baked into WebGL 2, and the BCD entry is api.WebGL2RenderingContext.drawBuffers.

That is supported on Android, which makes it seem a bit implausible that this extension couldn't be supported on Android.

So further digging still required.